(<class 'IsADirectoryError'>, IsADirectoryError(21, 'Is a directory'), <traceback object at 0x7f3e18099c00>)
[<FrameSummary file main.py, line 8 in <module>>, <FrameSummary file /home/runner/Microsystems/system.py, line 185 in start>, <FrameSummary file /home/runner/Microsystems/filesystem.py, line 13 in readFile>]